chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -224,10 +224,13 @@ def self.lookup_interface_info(node, conduit, intf_to_if_map = nil)
       end
 
       class Network
-        attr_reader :name, :address, :broadcast, :mac, :netmask, :subnet, :router, :usage, :vlan, :use_vlan, :interface, :interface_list, :add_bridge, :conduit
+        attr_reader :name, :address, :mtu, :broadcast, :mac, :netmask,
+          :subnet, :router, :usage, :vlan, :use_vlan, :interface,
+          :interface_list, :add_bridge, :conduit
         def initialize(net, data, rintf, interface_list)
           @name = net
           @address = data["address"]
+          @mtu = data["mtu"]
```

Comment:
Why not directly `data["mtu"] || 1500` here? (btw, I'd do `(data["mtu"] || 1500).to_i)` ;-))